charts/opensearch-operator/templates/opensearch-operator-controller-manager-deployment.yaml

```diff
@@ -43,6 +43,8 @@ spec:
         image: "{{ .Values.manager.image.repository }}:{{ .Values.manager.image.tag | default .Chart.AppVersion }}"
         name: operator-controller-manager
         imagePullPolicy: "{{ .Values.manager.image.pullPolicy }}"
+        securityContext:
```

Comment:
The securityContext field already exists below. I don't think we need to create another one. Just add the value that already exists in the values.yaml